client/dashboard/me/blocked-sites/index.tsx

```diff
@@ -0,0 +1,131 @@
+import { useMutation, useQuery } from '@tanstack/react-query';
+import { ExternalLink, Icon } from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { DataViews } from '@wordpress/dataviews';
+import { createInterpolateElement } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import { closeSmall } from '@wordpress/icons';
+import { store as noticesStore } from '@wordpress/notices';
+import { blockedSitesQuery, unblockSiteMutation } from '../../app/queries/me-blocked-sites';
+import DataViewsCard from '../../components/dataviews-card';
+import InlineSupportLink from '../../components/inline-support-link';
+import { PageHeader } from '../../components/page-header';
+import PageLayout from '../../components/page-layout';
+import type { BlockedSite } from '../../data/me-blocked-sites';
+
+const getHostname = ( url: string ) => {
+	try {
+		return new URL( url ).hostname;
+	} catch ( e ) {
+		return url;
+	}
+};
+
+export default function BlockedSites() {
+	// TODO: Implement infinite scroll once DataViews is updated to support it.
+	const { data: { sites, page } = { sites: [], page: 1 }, isLoading } = useQuery(
+		blockedSitesQuery()
+	);
+	const unblockSite = useMutation( unblockSiteMutation() );
+	const { createSuccessNotice, createErrorNotice } = useDispatch( noticesStore );
+
+	return (
+		<PageLayout
+			size="small"
+			header={
+				<PageHeader
+					title={ __( 'Blocked Sites' ) }
+					description={ createInterpolateElement(
+						__(
+							'Blocked sites will not appear in your Reader and will not be recommended to you. <link>Learn more</link>'
+						),
+						{
+							link: (
+								<InlineSupportLink
+									supportPostId={ 32011 }
+									// eslint-disable-next-line wpcalypso/i18n-unlocalized-url
+									supportLink="https://wordpress.com/support/reader/#interact-with-posts"
+								/>
+							),
+						}
+					) }
+				/>
+			}
+		>
+			<DataViewsCard>
+				<DataViews< BlockedSite >
+					data={ sites }
+					fields={ [
```

Comment:
Can we move the fields outside of the components? They don’t appear to depend on anything within the component.